UKHomeOffice/nmsw-ui
# NMSW-460 Add fallback if invalid alphacode added to FAL 1 & lookup country fails

# Ticket

NMSW-460

----

## AC

Bug - CYA page was stuck in 'loading' state
Identified this happened when a FAL 1 was uploaded with an invalid alphaCode (HON)

----

## To test

- create a voyage
- add a FAL 1 with an invalid alphaCode in FlagState e.g. HON
- continue on to check your answers page
- page should load with FlagState shown as 'HON'
- create a new voyage
- add a FAL 1 with a valid alphaCode in FlagState e.g. AUS
- continue on to check your answers page
- page should load with FlagState showing country name e.g. 'Australia'

----

## Notes
